test/vanilla/ignite-new.test.ts

```diff
@@ -16,20 +16,15 @@ afterEach(() => {
 })
 
 describe("Checking for ignite. 🪔", () => {
-  test(`ignite new (no name)`, async () => {
-    const result = await runError(`new`)
-    expect((result as any).stdout).toContain(`Project name is required`)
-  })
```

Comment:
[It's not possible for project name to be blank anymore](https://github.com/infinitered/ignite/pull/1999/files#diff-7920917860492dfa617b4e6b53368ad95f26db81b3edb99a38336c578d56971bR12-R17), so this test will hang forever and isn't a valid test case anymore